### ISSUE TITLE
Error while executing onJobBucketChange.functions

### ISSUE THREAD

**Issue by dollyagarwal** [google/voice-builder#33]
Voice-builder has been deployed to the GCP project and a job is created from the dashboard. But the status of the job stays at Exporting Resources. Upon checking the logs, I saw that there was an error while executing the onJobBucketChange funciton. PFB the screenshot:

![image](https://user-images.githubusercontent.com/49748410/72045713-cf47bb00-32dc-11ea-9e92-295a29447cd4.png)

Moreover, on further debugging, I found that pipelineInfo.json is not created for the job. The files in the job bucket are as shown in the screenshot below:

![image](https://user-images.githubusercontent.com/49748410/72045831-1170fc80-32dd-11ea-9b4e-3f74c00b75fa.png)

I am not sure, what is going wrong? I appreciate any help provided.

**Comment by pasindud** [comment]
Hi
can you add a ```console.log(object)``` to https://github.com/google/voice-builder/blob/master/cloud_functions/functions/index.js#L35

then re-ran the voice building, and check the logs again.

**Comment by dollyagarwal** [comment]
Hi, Thanks for your response. I added the console.log(object), here's the log:

![image](https://user-images.githubusercontent.com/49748410/72716611-87f0e280-3b98-11ea-8dfe-2eb1e5b35088.png)

**Comment by pasindud** [comment]
After the change did you run voice building again ?

**Comment by dollyagarwal** [comment]
Yes I did. I think the undefined displayed in the log is the object.

**Comment by pasindud** [comment]
In which case instead of logging object can you do ```console.log(event)```

**Comment by dollyagarwal** [comment]
Here's the screenshot of the log:

![screenshot](https://user-images.githubusercontent.com/49748410/72807724-07a4ad00-3c7e-11ea-976e-a7d2f0f77c17.png)

**Comment by thejaigoyal** [comment]
Hey @dollyagarwal did you get the solution for this, I'm stuck on the same issue

**Comment by crazzywizard** [comment]
Hey @dollyagarwal I changed `event.data` in line 34 of index.js to just `event` and was able to resolve these errors.
